app/routers/v1/router_api_v1.py

```diff
@@ -21,7 +21,7 @@ def reload_model_api_v1(func):
     @wraps(func)
     def wrapper(*args, **kwargs):
         global novel_corona_api, dt, ts
```

Comment:
Would you mind changing **novel_corona_api** to **COVID_API_V1** in this file as well?

Reply:
done